.github/workflows/pull_request.yml

```diff
@@ -164,3 +164,9 @@ jobs:
       run: go install honnef.co/go/tools/cmd/staticcheck@latest
     - name: Run staticcheck
       run: GOGC=30 staticcheck ./...
+    - name: Get Ondatra linter
+      run: go install github.com/openconfig/ondatra/gnmi/migrator@latest
+    - name: Run Ondatra linter
+      run: |
+        migrator -json ./feature/... > lint.json
```

Comment:
Can we change the lint2annotation to read from stdin? Then we can just:

migrator -json ./feature/... | go run ./tools/lint2annotation